src/ProtocolHandler/DataParser/Request.cs

```diff
@@ -0,0 +1,76 @@
+using System.Text;
+using Chorizo.ProtocolHandler.HttpHeaders;
+
+namespace Chorizo.ProtocolHandler.DataParser
```

Comment:
Request probably doesn't belong in this namespace.

Reply:
Added to Exchange namespace based on this [StackOverflow](https://stackoverflow.com/a/325358) post.